Bottom: 633ed7bb6ad3ac90293fdf4cec7528f8a82b6d9a
Top:    624244a770d982c112a7c80139c210334b201e07
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 19:22:05 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6b52e57..ff8a42b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1553,9 +1553,9 @@ ghid_draw_everything (BoxTypePtr drawn_area)
   side = SWAP_IDENT ? SOLDER_LAYER : COMPONENT_LAYER;
 
   /* Draw pins, pads, vias below silk */
-  if (global_view_2d &&
-      gui->set_layer (SWAP_IDENT ? "bottomsilk" : "topsilk",
-                      SWAP_IDENT ? SL (MASK, BOTTOM) : SL (MASK, TOP), 0)) {
+  if (global_view_2d) {
+    gui->set_layer (SWAP_IDENT ? "bottomsilk" : "topsilk",
+                    SWAP_IDENT ? SL (MASK, BOTTOM) : SL (MASK, TOP), 0);
 
     /* Mask out drilled holes */
     hidgl_flush_triangles (&buffer);
